packages/dev/core/src/Misc/snapshotRenderingHelper.ts

```diff
@@ -134,6 +141,29 @@ export class SnapshotRenderingHelper {
             return;
         }
 
+        if (this._disableRenderingRefCount === 0) {
+            // Snapshot rendering switches from enabled to disabled
+            // We reset the performance priority mode to that which it was before enabling snapshot rendering, but first set it to “BackwardCompatible” to allow the system to regenerate resources that may have been optimized for snapshot rendering.
+            // We'll then restore the original mode at the next frame.
+            this._scene.performancePriority = ScenePerformancePriority.BackwardCompatible;
+            if (this._currentPerformancePriorityMode !== ScenePerformancePriority.BackwardCompatible) {
+                this._pendingCurrentPerformancePriorityMode = this._currentPerformancePriorityMode;
+                this._scene.executeWhenReady(() => {
+                    this._executeAtFrame(
+                        this._engine.frameId + 2,
+                        () => {
+                            // if this._disableRenderingRefCount === 0, snapshot rendering has been enabled in the meantime, so do nothing

```

Comment:
```suggestion
                            // if this._disableRenderingRefCount === 0 snapshot rendering has been enabled in the meantime, so do nothing
```

forcing a new build (not really needed)